webpack/assets/javascripts/react_app/common/colors.scss

```diff
@@ -1,7 +1,12 @@
+// colors
 $pf-black-200: #ededed;
 $pf-red-200: #a30000;
 $pf-red-100: #c00;
 $pf-orange-400: #ec7a08;
 $pf-gold-400: #f0ab00;
 $pf-green-400: #3f9c35;
 $pf-green-600: #1e4f18;
+
+$switcher-max-height: 300px;
+$switcher-max-width: 200px;
+$switcher-min-width: 200px;
```

Comment:
I prefer to keep it as it so it will be easier for me to do the rebase here later https://github.com/theforeman/foreman/pull/5212

I am trying to fully address this issue in this #5212